CRM/Contact/Form/Search/Custom/ContribSYBNT.php

```diff
@@ -43,7 +43,7 @@ class CRM_Contact_Form_Search_Custom_ContribSYBNT extends CRM_Contact_Form_Searc
    * @param $formValues
    */
   public function __construct(&$formValues) {
-    $this->_formValues = $formValues;
+    $this->_formValues = self::formatSavedSearchFields($formValues);
```

Comment:
works if you change it to     self::formatSavedSearchFields($formValues);
$this->_formValues = $formValues;

Reply:
I thought so but I encountered an error at one place (can't recall the use-case). Should I still revert this change?